docs/developer-docs/latest/update-migration-guides/migration-guides/v4/snippets/codemod-prerequisites.md

```diff
@@ -0,0 +1,5 @@
+To use the codemods for automatic migration, the [Strapi codemods repository](https://github.com/strapi/codemods) should be cloned by running the following command in a terminal:
+
+```bash
+git clone https://github.com/strapi/codemods.git
+```
```

Comment:
We no longer need to clone since we will use an npx command so this can be removed